segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentStoreMetrics.java

```diff
@@ -37,9 +37,7 @@
 import java.util.concurrent.TimeUnit;
 import lombok.val;
 
-import static io.pravega.shared.MetricsTags.containerTag;
-import static io.pravega.shared.MetricsTags.eventProcessorTag;
-import static io.pravega.shared.MetricsTags.throttlerTag;
+import static io.pravega.shared.MetricsTags.*;
```

Comment:
This kind of imports are not allowed.

Reply:
corrected...didnt come up in checkstyle though...doesnt checkstyle catch this?

Reply:
Yes, it should have been detected by checkstyle.

Reply:
Strange to see that this was not caught by the checkstyle. otherwise such things are always handled during the build part itself.